app/index.sass

```diff
@@ -30,6 +30,10 @@ body
 .container-narrow > hr
   margin: 30px 0
 
+svg
+  width: 100%
+  height: 100%
+
```

Comment:
Yeah I think the SVG might always be able to have full width/height, we just need to resize intelligently according to its parent divs.